src/views/ReviewConfirm.vue

```diff
@@ -1,5 +1,14 @@
 <template>
   <div>
+    <section class="mt-10 benefit-company-statement" v-if="isTypeBcomp">
+        <header>
```

Comment:
Not sure, but might be an extra space here?

Reply:
Fixed.